node/node.go

```diff
@@ -374,27 +355,60 @@ func (node *Node) AddPendingReceipts(receipts *types.CXReceiptsProof) {
 		Msg("Got ONE more receipt message")
 }
 
-func (node *Node) startRxPipeline(
-	receiver p2p.GroupReceiver, queue *msgq.Queue, numWorkers int,
-) {
-	// consumers
-	for i := 0; i < numWorkers; i++ {
-		go queue.HandleMessages(node)
+// Start kicks off the node message handling
+func (node *Node) Start() error {
+	allTopics := node.host.AllTopics()
```

Comment:
Where are the topics registered?

Reply:
They are already registered done by this point, same as before

Reply:
Ok, So we don't have topic specific worker pool anymore? Everything is competing for the same resources with equal weight?

Reply:
No, each one is having its own channel - same as before conceptually

Reply:
see the for loop

Reply:
ok, cool